mmcv/ops/merge_cells.py

```diff
@@ -150,7 +154,10 @@ def _binary_op(self, x1, x2):
 
 class GlobalPoolingCell(BaseMergeCell):
 
-    def __init__(self, in_channels=None, out_channels=None, **kwargs):
+    def __init__(self,
+                 in_channels: Optional[int] = None,
+                 out_channels: Optional[int] = None,
```

Comment:
```suggestion
                 in_channels: int,
                 out_channels: int,
```

The default value can be removed, which should be a bug. @jshilong have a look.

Reply:
in_channels or out_channelscan be None when with_out_conv is False so Optional[int] is correct.
I create a [pr](https://github.com/open-mmlab/mmcv/pull/2062) to ensure in_channels or out_channels can not be None when with_out_conv is True.